core/data/src/main/java/com/iteneum/ItemList.kt

```diff
@@ -0,0 +1,7 @@
+package com.iteneum
+
+data class ItemList(
+    var urlImage: String? = null,
+    var title: String?,
+    var description: String?
```

Comment:
Also, if all the data class variables could be null so please define a default variable at the end to avoid the null pointer exceptions (like you did in the first variable).